setup_cfg_fmt.py

```diff
@@ -515,20 +539,31 @@ def _ver_type(s: str) -> Version:
         return version
 
 
+class ImplicitlySetVersion(Version):
+    def __new__(self, x: int, y: int) -> ImplicitlySetVersion:
+        return tuple.__new__(ImplicitlySetVersion, (x, y))
```

Comment:
does this do anything?  we can probably use `None` as a sentinel instead

Reply:
That changes the `help` output though

Reply:
I guess I could make custom `--help` output?

Reply:
yeah can probably just format in the actual default

Reply:
I think there is actually no need to set a default `min-py-version` anymore because the argument is totally optional. Lmk